src/components/Header/Header.tsx

```diff
@@ -9,11 +9,161 @@ const navItems: INavItem[] = [
     id: uid(),
     name: 'Products',
     link: '/products',
+    content: [
+      {
+        id: uid(),
+        name: 'Consumer',
+        link: '/consumer',
+      },
+      {
+        id: uid(),
+        name: 'Professional',
+        link: '/professional',
+      },
+      {
+        id: uid(),
+        name: 'Enterprise',
+        link: '/enterprise',
+      },
+    ],
   },
   {
     id: uid(),
     name: 'Services',
     link: '/services',
+    content: [
+      {
+        id: uid(),
+        name: 'Services',
+        link: '/services',
+        subContent: [
+          {
+            id: uid(),
+            name: 'Drone Repairs',
+            link: '/services/drone-repairs',
+          },
+          {
+            id: uid(),
+            name: 'Drone Maintenance',
+            link: '/services/drone-maintenance',
+          },
+          {
+            id: uid(),
+            name: 'Drone Software',
+            link: '#',
+          },
+          {
+            id: uid(),
+            name: 'Drone Consulting',
+            link: '/services/drone-consulting',
+          },
+          {
+            id: uid(),
+            name: 'Drone Detection',
+            link: '/services/drone-detection',
+          },
+          {
+            id: uid(),
+            name: 'Drone Rentals',
+            link: '/services/drone-rentals',
+          },
+          {
+            id: uid(),
+            name: 'Drone Research',
+            link: '/services/drone-applied-research',
+          },
+          {
+            id: uid(),
+            name: 'Drone Education',
+            link: '/services/drone-education',
+          },
+          {
+            id: uid(),
+            name: 'Custom Solutions',
+            link: '/services/custom-solutions',
+          },
+        ],
+      },
+      {
+        id: uid(),
+        name: 'Software Solutions',
+        link: '#',
+        subContent: [
+          {
+            id: uid(),
+            name: 'DJI Terra',
+            link: '#',
```

Comment:
These Products should come from Sanity and they should navigate to their respective pages using the slug.

**Example** 
https://dronehubgh.com/products/dji-terra
https://dronehubgh.com/products/dji-flight-simulator